modeling-cmds/src/def_enum.rs

```diff
@@ -279,6 +279,21 @@ define_modeling_cmd_enum! {
             pub rotate_duplicates: bool,
         }
 
+        /// Create a helix using the input cylinder and other specified parameters.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ModelingCmdVariant)]
+        pub struct EntityMakeHelix {
+            /// ID of the cylinder.
+            pub cylinder_id: Uuid,
+            /// Number of revolutions.
+            pub revolutions: f64,
+            /// Start angle (in degrees).
+            pub start_angle: f64,
```

Comment:
Let's not use degrees here, instead, let's use the `Angle` type so users can specify either degrees or radians (or other types in the future). Don't forget to update the docstring too.